lib/helpers.ts

```diff
@@ -217,3 +219,8 @@ export function fill(value: string, times: number): string[]{
 
 	return repeatedValues;
 }
+
+export function defaultProfileDir(): string {
+	var cacheRootDirectoryPath = hostInfo.isWindows() ? process.env.LocalAppData : path.join(osenv.home(), ".local/share");
+	return path.join(cacheRootDirectoryPath, "Telerik", "BlackDragon", ".appbuilder-cli");
```

Comment:
Can we rename .appbuilder to appbuilder? It is in an already hidden dir, so there is no need for it to be hidden as well.